python/lsst/ap/association/diaCalculationPlugins.py

```diff
@@ -84,6 +88,106 @@ def calculate(self, diaObject, diaSources, **kwargs):
         else:
             diaObject["ra"] = aveCoord.getRa().asDegrees()
             diaObject["decl"] = aveCoord.getDec().asDegrees()
+            diaObject["radecTai"] = np.max(diaSources["midPointTai"])
+
+
+class HTMIndexDiaPositionConfig(DiaObjectCalculationPluginConfig):
+
+    indexer = IndexerRegistry.makeField(
+        doc='Select the spatial indexer to use within the database.',
+        default='HTM'
```

Comment:
While HTM is a fine default here, you've hardwired "HTM" into the plugin names here. Are there situations where spatial indexers other than HTM might be desired? If so, could you call this something more generic like `SpatialIndexDiaPosition`?

Reply:
That would be done by a different plugin.

Reply:
OK, it's a little odd to me then that you have a config option to select the spatial indexer, which implies a different choice could be made. But I don't have a suggestion for a better way to do it. Maybe a check to make sure `indexer == 'HTM'` ?